src/app/_components/BlogPostArchiveButton/index.tsx

```diff
@@ -0,0 +1,14 @@
+export default function ArchiveButton({ collection }: { collection: string }) {
+  function formatArchiveButton(text: string): string {
+    // TODO: Extend to format talks-and-roundtables to Talks & Roundtables
+    // TODO: Extend to fromat case-studies to Case Studies
+
+    return text.charAt(0).toLowerCase() + text.slice(1)
```

Comment:
What is this for, to turn the word into lower case?

Reply:
it's a converter to convert what ArchiveButton receives and make a compatible key of `collections`

Reply:
This is the same as just `text.toLowerCase()` though